src/appimagetool.c

```diff
@@ -903,6 +911,28 @@ main (int argc, char *argv[])
                         printf("Will not guess update information since zsyncmake is missing\n");
                     }
                 }
+	    } else if(GITHUB_REPOSITORY){
+		gchar *zsyncmake_path = g_find_program_in_path ("zsyncmake");
+		if (zsyncmake_path){
+		   char buf[1024];
+		   printf("Running on GitHub Actions\n");
+		   if (strstr(GITHUB_REF, "/pull/")) {
+			printf("Will not calculate update information for GitHub because this is a pull request\n");
+		   } else {
+                        gchar **parts = g_strsplit (GITHUB_REPOSITORY, "/", 0);
+
+			char* channel;
+			if (GITHUB_REF != "" && strstr(GITHUB_REF, "/continuous/")) {
```

Comment:
You never check that `GITHUB_REF` is set at all.

Reply:
Done on https://github.com/AppImage/AppImageKit/pull/1075/commits/34bab4e64de8dce05c25e3fe2f01012f0e0ab74e